### PR TITLE
DOCSP-40216 Add search index management limitations compass

### PR DESCRIPTION
**Pull request by sonderdonk-mdb**
## DESCRIPTION
added a bullet for limitations to Compass documentation and a Tip

## STAGING
https://preview-mongodbsonderdonkmdb.gatsbyjs.io/compass/DOCSP-40216/indexes/#limitations

## JIRA
https://jira.mongodb.org/browse/DOCSP-40216

## BUILD LOG
https://workerpool-boxgs.mongodbstitch.com/pages/job.html?collName=queue&jobId=6679cb54ebce19f314b186fe

## Self-Review Checklist

- [x] Is this free of any warnings or errors in the RST?
- [x] Is this free of spelling errors?
- [x] Is this free of grammatical errors?
- [x] Is this free of staging / rendering issues?
- [x] Are all the links working?

## External Review Requirements

[What's expected of an external reviewer?](https://wiki.corp.mongodb.com/display/DE/Reviewing+Guidelines+for+the+MongoDB+Server+Documentation)

### CODE REVIEW

File: source/indexes.txt [hunk 1]
```diff
@@ -173,6 +173,18 @@ Limitations
 - The :guilabel:`Indexes` tab is not available if you are connected 
   to a :atlas:`Data Lake </data-lake>`. 
 
+- You can only manage Atlas Search indexes in Compass if your deployment is
```
Comment (sarah-olson-mongodb) [inline source/indexes.txt:176]: (nit -- [positive statements](https://www.mongodb.com/docs/meta/style-guide/writing/use-positive-statements/))
You could omit "only" here if you like. The sentence will mean the same thing, but is more positive.

File: source/indexes.txt [hunk 2]
```diff
@@ -173,6 +173,18 @@ Limitations
...
+  local, has an Atlas cluster tier of M10 or larger, and runs MongoDB 7.0 
+  or higher. 
+  
+  .. tip::
+
+     For clusters running an earlier version of MongoDB or 
+     :ref:`shared tier clusters <shared-tier-cluster>`, 
+     you can manage your Atlas Search indexes from the Atlas UI, with the 
+     :ref:`Atlas CLI <atlas-cli>`, or with the 
+     :ref:`Atlas Administration API <atlas-admin-api>`.
```
Comment (sarah-olson-mongodb) [inline source/indexes.txt:186]: (nit -- wording)
This is subjective and optional. Consider a slight tweak to create more parallelism. 
```suggestion
     you can manage your Atlas Search indexes using the Atlas UI, the 
     :ref:`Atlas CLI <atlas-cli>`, or the 
     :ref:`Atlas Administration API <atlas-admin-api>`. 
```

**Review by sarah-olson-mongodb — APPROVED** [review]

**Review by sarah-olson-mongodb — APPROVED** [review]
Thanks @sonderdonk-mdb! LGTM with a couple of minor changes for your consideration.

File: source/indexes.txt (hunk 2 above)
Comment (sonderdonk-mdb) [reply source/indexes.txt]: Love parallelism, changed!

File: source/indexes.txt (hunk 1 above)
Comment (sonderdonk-mdb) [reply source/indexes.txt]: Thanks for the edit, just fixed!

**Review by betsybutton — APPROVED** [review]
Thanks for getting this up so quickly, Sarah!

**Pull request merged**